homeassistant/components/geniushub/switch.py

```diff
@@ -1,13 +1,29 @@
 """Support for Genius Hub switch/outlet devices."""
+from datetime import timedelta
+
+import voluptuous as vol
+
 from homeassistant.components.switch import DEVICE_CLASS_OUTLET, SwitchEntity
+from homeassistant.const import ATTR_ENTITY_ID
+from homeassistant.helpers import config_validation as cv, entity_platform
 from homeassistant.helpers.typing import ConfigType, HomeAssistantType
 
-from . import DOMAIN, GeniusZone
-
-ATTR_DURATION = "duration"
+from . import ATTR_DURATION, DOMAIN, GeniusZone
 
 GH_ON_OFF_ZONE = "on / off"
 
+SVC_SET_SWITCH_OVERRIDE = "set_switch_override"
+
+SET_SWITCH_OVERRIDE_SCHEMA = vol.Schema(
```

Comment:
If we pass a dict with validators to the entity service helper it will generate an entity service schema for us.

https://developers.home-assistant.io/docs/dev_101_services#entity-services